CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 2.3.0
+
+### Major changes
+* `bem-core` dependency was updated to [2.7.0](https://github.com/bem/bem-core/blob/v2.7.0/CHANGELOG.ru.md#270).
```

Comment:
Previously [we gave the GitHub link](https://github.com/bem/bem-components/blob/v2/CHANGELOG.md#major-changes). Are you sure that we should deviate from this now?